### ISSUE TITLE
Redirect /beniz/seeks to /Seeks/seeks

### ISSUE THREAD

**Issue by GreenLunar** [beniz/seeks#27]
Please redirect this repository to `/Seeks/seeks` and let new developers continue this extraordinary project.

**Comment by beniz** [comment]
/seeks already exists, right ?

Now, this project is to complicated, old and ill-structured for a reboot or even maintenance. At least this is my point of view...

What would you think of starting an open discussion on how to move forward, including how to boot up from scratch ?

I do believe distributed, privacy-oriented, search is still unsolved. Therefore it does require attention. But as anything distributed, it is kind of a community problem, and it would be best to hear from everyon e interested over a period of a few months, no ?

**Comment by GreenLunar** [comment]
> /seeks already exists, right ?

`/seeks-project` is available, but perhaps `/seeks` would be kind enough to give her/his domain to this project.

> Now, this project is to complicated, old and ill-structured for a reboot or even maintenance. At least this is my point of view...

Maybe other would like to refactor it and they might prefer to start with Seeks as it is, because it has substance and it is possible to compile it and see it in action.

> What would you think of starting an open discussion on how to move forward, including how to boot up from scratch ?

I like this idea.

> I do believe distributed, privacy-oriented, search is still unsolved. Therefore it does require attention. But as anything distributed, it is kind of a community problem, and it would be best to hear from everyon e interested over a period of a few months, no ?

Maybe so, maybe not; either way, I think we should provide an official domain (a GitHub organization) to archive this repository and have an ability to join others to the official repository.

**Comment by beniz** [comment]
> Maybe so, maybe not; either way, I think we should provide an official domain (a GitHub organization) to archive this repository and have an ability to join others to the official repository.

The low number of PR (well, one...) shows that no developper is active here. Are you a developper btw ?

This is the reason why I highly suggest and am ready to help building a discussion space that would help design the futur path(s).

**Comment by GreenLunar** [comment]
> The low number of PR (well, one...) shows that no developper is active here. Are you a developper btw ?

No, I am not, unfortunately.  The number of PR is low because of lack of popularity.

> This is the reason why I highly suggest and am ready to help building a discussion space that would help design the futur path(s).

To me, it appears like developers are more likely to put attention to archived contents and are not likely have patient for discussions; maybe it would be better to combine both.

**Comment by silverhook** [comment]
I’m not a dev either (esp. not on this level of expertise!), but Seeks to this day remains my favourite search “engine”/system.

Other than by some publicity, translations or perhaps even some donation, I cannot help out, I’m afraid :/

**Comment by GreenLunar** [comment]
I just read matija.suklje.name.
I am a lawyer, too :)

**Comment by silverhook** [comment]
[OT]
@GreenLunar, high five! ,o/
[/OT]

**Comment by GreenLunar** [comment]
@Seeks, we would like to [create an organization](https://github.com/organizations/new) named Seeks; may you please hand over your username?

**Comment by Quix0r** [comment]
I have reported the user to github, let's see if they delete it as the user behind seems to be entirely inactive and is only blocking the name for this organization. Stay tuned, maybe something will happen.

**Comment by silverhook** [comment]
@beniz Have you thought about the future of Seeks or of a new distributed search system by now? I really miss Seeks and I think the world is ripe now for something like Seeks finally.

As mentioned, I’m not a coder, but would be happy to throw in some money and help with finding some funding (e.g. NLnet, Mozilla Grant, etc.).

**Comment by Quix0r** [comment]
Yes, /Seeks/seeks gives a 404 page, so the user is gone. Go, register it and push all over there.
